addons/mass_mailing/models/ir_model.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+
+
+class IrModel(models.Model):
+    _inherit = 'ir.model'
+
+    is_mailing_enabled = fields.Boolean(
+        string="Mailing Enabled",
+        compute='_compute_is_mailing_enabled', search='_search_is_mailing_enabled',
+        help="Whether this model supports marketing mailing capabilities (notably email and SMS).",
+    )
+
+    def _compute_is_mailing_enabled(self):
+        for model in self:
+            model.is_mailing = getattr(self.env[model.model], '_mailing_enabled', False)
```

Comment:
Hum... :D

That's this moment when you blame the guy that asked you to rename ^^